src/app/_components/Footer/Footer.js

```diff
@@ -1,5 +1,101 @@
-import React from 'react';
+"use client";
+
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import OutlinedBtn from "../OutlinedBtn";
+import * as FooterImg from "../../../../public/assets/icons/white-logo.svg";
+import InstagramComponent from "../Icons/Instagram";
+import FacebookComponent from "../Icons/Facebook";
+import XComponent from "../Icons/XSocial";
```

Comment:
🟠 can these imports be prettified with import aliases?

Reply:
added some aliases